### ISSUE TITLE
Add maxLines parameter to FlowRow

### ISSUE THREAD

**Issue by ARTI1208** [JetBrains/compose-multiplatform#3785]
Our team is looking forward to move from Android/IOS views to compose. Currently we have custom FlowRow views for each platform. Compose's FlowRow seems to have what we need except for maxLines setting. For now I just copypasted huge amount of FlowLayout code to implement this feature, but I want to see this option in upstream. I've checked https://github.com/JetBrains/compose-multiplatform-core repo and README.md and CONTRIBUTING.md say that compose/foundation folder is not the one to be contributed to via GitHub. However this comes from original androidx repo so I'm unsure whether this applies to compose-multiplatform-core

TLDR: Please help me understand what should I do to get maxLines implemented in upstream FlowRow (whether by myself or by someone from jb/google compose teams)

**Comment by pjBooms** [comment]
As it seems to be general enhancement that affects Jetpack Compose too, I would recommend to contribute it directly to AndroidX repository or create the same enhancement in Jetpack Compose issue tracker and discuss it there with the Google team.

Once it appears in Jetpack Compose, it will be merged to Compose Multiplatform as well one day.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-3785">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
